utils/docker/Dockerfile.focal

```diff
@@ -30,7 +30,7 @@ RUN mkdir /ms-playwright && \
     cd /ms-playwright-agent && \
     dotnet new console && \
     echo '<?xml version="1.0" encoding="utf-8"?><configuration><packageSources><add key="local" value="/tmp/"/></packageSources></configuration>' > nuget.config && \
-    dotnet add package Microsoft.Playwright && \
+    dotnet add package Microsoft.Playwright --prerelease && \
```

Comment:
when our current version was a e.g. next-1 version it didn't install the expected version inside Docker. This fixes it.